kratos/geometries/nurbs_surface_geometry.h

```diff
@@ -380,8 +358,8 @@ class NurbsSurfaceGeometry : public Geometry<typename TContainerPointType::value
     }
 
     /* Provides all knot span intervals of the surface in u-direction.
-    @return vector of knot span intervals.
-    */
+     * @return vector of knot span intervals.
+     */
     std::vector<NurbsInterval> KnotSpanIntervalsU() const
```

Comment:
Return Ref?

Reply:
this cannot be returned by ref as it is created within this function

Reply:
to be honest we should sooner or later get rid of the knot span intervals anyways...

Reply:
Well yes that makes sense.. But I guess you are right. We should probably deal with this differently.